coralproject/talk
# [CORL-2527]: show status dropdown for moderator and staff roles

## What does this PR do?

These changes make it so that the status dropdown is shown for moderator (site and org moderators) and staff users, as well as commenters.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Be logged in as either an admin or a site moderator, test both.

 You can go to the `Community` tab and find a staff user. See that they have the dropdown in the status column. Click the dropdown and see that all options are enabled.
 
 Go to the `Community` tab and find a site moderator user. See that they have the dropdown in the status column. Click the dropdown and see that all options are enabled. If you click ban, you should **not** see the `All sites` option to ban them across all sites. If you ban them on a site that they don't moderate, you should see that they are banned. If you try to ban them on a site that they do moderate, you should see an error message and you should **not** be able to ban them on that site.
 
 Go to the `Community` tab and find an org moderator user. See that they have the dropdown in the status column. Click the dropdown and see that the `Manage Ban` options is disabled but other options are enabled.
 
 Confirm that admins do **not** have the status dropdown.
 
 You shouldn't see a status dropdown for the user you are logged in as.
 
## How do we deploy this PR?
